package.json

```diff
@@ -9,23 +9,29 @@
     "coverage": "yarn codecov"
   },
   "dependencies": {
+    "@types/lodash.uniq": "^4.5.5",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
No need for isomorphic-fetch if we're just on the backend side?

Reply:
Switched to `node-fetch`, thanks for pointing it out!